bliss/models/galaxy_net.py

```diff
@@ -209,6 +209,9 @@ def optimizer_step(
         if optimizer_idx == 1:
             if self.trainer.global_step > self.residual_delay_n_steps:
                 optimizer.step(closure=optimizer_closure)
+            else:
+                # call closure by itself to run `training_step` + `backward` without optimizer step
+                optimizer_closure()
```

Comment:
I'm a little confused by this change. Why would we want to run the `training_step` and `backward` on the residual autoencoder before we start optimizing it?